Makefile

```diff
@@ -7,6 +7,7 @@ DIST_DIR := _dist
 GO_PACKAGES := action chart config dependency log manifest release plugins/sec plugins/example codec
 MAIN_GO := helm.go
 HELM_BIN := $(BIN_DIR)/helm
+PATH_WITH_HELM = PATH="$(shell pwd)/$(BIN_DIR):$(PATH)"
```

Comment:
k. I guess either is fine, I suggested `${PWD}` because afaik it doesn't invoke the shell